lib/src/css_class_set.dart

```diff
@@ -33,7 +33,7 @@ class ElementCssClassSet extends CssClassSetImpl {
 }
 
 /** A Set that stores the CSS class names for an element. */
-abstract class CssClassSet implements Set<String> {
+abstract class CssClassSet extends SetBase<String> {
```

Comment:
I'm not sure these are in the right places, but I could be wrong.  I think it looks like:

abstract class CssClassSet ....

abstract class CssClassSetImpl implements CssClassSet

class ElementCssClassSet extends CssClassSetImpl

So I think the new methods needs to go on either CssClassSetImpl or ElementCssClasSet

I think best would be making CssClassSetImpl extend SetBase<String> implements CssClassSet?

Or am I confused?

Reply:
you are right.

Kevin has a [PR](https://github.com/dart-lang/html/pull/58) which cleans up the package along with these changes. So going to go with that and discard this one.